docs/layouts/shortcodes/generated/expert_rest_section.html

```diff
@@ -20,6 +20,12 @@
             <td>Long</td>
             <td>The time in ms that the client waits for the leader address, e.g., Dispatcher or WebMonitorEndpoint</td>
         </tr>
+        <tr>
+            <td><h5>rest.cache.execution-graph.expiry</h5></td>
```

Comment:
Will this config be meaningful to end users? Which REST endpoints does this control? For example, it is not intuitive that this controls the checkpoint list API